packages/react-devtools-shared/src/backend/fiber/renderer.js

```diff
@@ -1997,7 +1885,12 @@ export function attach(
         3 + pendingOperations.length,
       );
       operations[0] = rendererID;
-      operations[1] = currentRootID;
+      if (currentRoot === null) {
+        // TODO: This is not always safe so this field is probably not needed.
+        operations[1] = -1;
```

Comment:
I refactored currentRoot to be the instance. I used the `(null: any)` since any time it is accessed it should be known to be initialized.

However, this revealed that there were a couple of place where this was read outside a context where it's set.

These used to be `-1`. That suggests that these fields are not actually used by the protocol since it is apparently resilient to not receiving the right value. Could be an opportunity to clean up.